### ISSUE TITLE
Needless return

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#1528]
The following `return` could be dropped due to it being the last return of function and will return `void`.

```php
function test() {
    if (mt_rand() === 0) {
        return;
    }
}
```

It is similar to the looping version:

```php
while (true) {
    if (mt_rand() === 0) {
        continue; // [EA] It's not really makes sense placing continue 
                  //      here as loop will continue from here anyway. 
    }
}
```

**Note:**

 * Should works only for `void` or undefined return type (eg. `function (): void` or just `function ()`);

**Comment by ea-inspections-team** [comment]
Thank you for the proposal @rentalhost! PhpStorm is already reporting this case, so I will close this issue.
